web/src/main/resources/org/openmrs/web/filter/initialization/databasetablesanduser.vm

```diff
@@ -271,16 +270,10 @@
         .button-group {
             flex-direction: column;
         }
-
         .button {
             width: 100%;
             justify-content: center;
         }
     }
 </style>
-
-<script type="text/javascript">
-    document.forms[0].elements[0].focus();
```

Comment:
Why remove this?

Reply:
Didn't note down why i did this 😆 so i will just put it back

Reply:
I have added it back and tested.